### ISSUE TITLE
Test loading from a file

### ISSUE THREAD

**Issue by egabancho** [inveniosoftware/invenio-saml#8]
`python3-saml` offers this as a possibility so we should be able to do things like

```
SSO_SAML_IDPS = {
    "idp-abc": {
        "settings_file_path": "./saml/idb-abc.json",
    },
    ...
}
```

The location of the file it is up to discussion.

(Extracted from #3)

**Comment by mb-wali** [comment]
By providing the settings_path in settings of idp,

**Comment by mb-wali** [comment]
This configuration should be achieved from the ```flask_sso_saml``` module.
[this](https://github.com/egabancho/flask-sso-saml/blob/02fe6c22f62903d617093760cd2250e04a6385f9/flask_sso_saml/ext.py#L197) line of code is expecting XML, [see here](https://github.com/onelogin/python3-saml/blob/6f8ca9c1f628db3a818312e678d5c2c108efbe9c/src/onelogin/saml2/idp_metadata_parser.py#L83). 
the function should be something like [this](https://github.com/onelogin/python3-saml/blob/6f8ca9c1f628db3a818312e678d5c2c108efbe9c/src/onelogin/saml2/settings.py#L229).

i tried to add the ```settings_file_path``` in my configuration, it does find the directory but it expect XML structure. 
the screenshot is the error page.
![screenshot](https://user-images.githubusercontent.com/44528277/73733567-839dfb00-473c-11ea-9a21-e5d6f6977271.PNG)

**Comment by mb-wali** [comment]
closed with #18
